doc/source/operations/rocky-linux-9.rst

```diff
@@ -456,6 +456,13 @@ address them.
 Once happy that the system has been restored to full health, move onto the next
 host or batch or hosts.
```

Comment:
The indentation suggests this part should be after the numbered list.

Reply:
I'd put this step after as we wouldn't want to enable it again until we are "happy that the system has been restored to full health". I'll edit to make this clearer

Reply:
How would we know it's been restored to full health if it's disabled?

Reply:
I interpreted this more as the migrated hypervisor(s) are configured and running all the services and no errors/alerts are firing.